src/app/modules/gb-side-panel-module/accordion-components/gb-tree-nodes/gb-tree-nodes.component.html

```diff
@@ -1,22 +1,9 @@
 <div id="tree-nodes-component">
 
   <div *ngIf="!isLoading">
-    <!--the search/filter input-->
-    <div>
-      <p-autoComplete [(ngModel)]="searchTerm"
-                      styleClass="gb-tree-filter-input"
-                      placeholder="filter"
-                      (onClear)="onFiltering($event)"
-                      (completeMethod)="onFiltering($event)"></p-autoComplete>
-      <span>&nbsp;</span>
-      <button class="btn btn-secondary btn-sm filter-btn"
-              (click)="clearFilter()"
-              type="button">clear
-      </button>
-      <div *ngIf="hits > 0" class="filter-result">
-        {{hits}} results found
-      </div>
-    </div>
+    <gb-tree-search
+      [tree]="treeNodeService.treeNodes"
+    ></gb-tree-search>
```

Comment:
I like the idea of abstracting it only for tree nodes, currently nicely separated from the category list, but also allow for future inclusion :)